app/jobs/clock_check_job.rb

```diff
@@ -0,0 +1,33 @@
+class ClockCheckJob < ApplicationJob
```

Comment:
a) This should probably live in https://github.com/ManageIQ/manageiq/blob/69f83f019abf043e6643680eae60a4e674f2390f/app/models/miq_schedule_worker/jobs.rb and not use ApplicationJob
b) I thought we already had an ntp check ... @jrafanie?